browse/services/listing/db_listing_impl.py

```diff
@@ -1,17 +1,16 @@
 """arXiv listings backed by a DB."""
```

Comment:
The DB implementation was changed to work with the refactor. But I'm thinking that we should go with the file based one since the DB one is incomplete and the tables it uses lack data for before 2010.